Makefile

```diff
@@ -168,13 +172,35 @@ docker-build:
 	docker build --tag ink-playground .
 
 docker-run:
-	docker run --publish 80:4000 ink-playground
+	docker run \
+	  --volume /tmp:/tmp \
+	  --volume /var/run/docker.sock:/var/run/docker.sock \
+	  --publish $(PORT):4000 \
```

Comment:
We should provide a default value for the PORT.

Reply:
:+1: